csharp/src/Ice/ITransceiver.cs

```diff
@@ -35,8 +35,9 @@ public interface ITransceiver : IDisposable
 
         /// <summary>Initializes the transceiver. The transceiver might use this method to establish or accept the
         /// connection.</summary>
+        /// <param name="connection">The connection that being initialize.</param>
```

Comment:
```suggestion
        /// <param name="connection">The connection being initialized.</param>
```

Reply:
The summary says "The transceiver might use this method to establish or accept the connection" but then the first parameter is a connection. I'm confused.

Reply:
This is called from the connection InitializeAsync, here the connection has been created but has not been initialized, I will try to clarify the comment in InitializeAsync